canonical-web-and-design/releases.demo.haus
# Add table sorting to the release table

## Done
- Upgraded to Vanilla 2.0.1
- Add the Vanilla sortable pattern to the table

## QA
- Wait for the deployed to staging to populate
- Then sort by clicking the table heading
- Check its in the correct order

Fixes https://github.com/canonical-web-and-design/releases.demo.haus/issues/12
